xray/commands/audit/generic/generic.go

```diff
@@ -93,21 +99,42 @@ func (auditCmd *GenericAuditCommand) CreateXrayGraphScanParams() *services.XrayG
 }
 
 func (auditCmd *GenericAuditCommand) Run() (err error) {
-	if err != nil {
-		return
-	}
 	auditParams := NewAuditParams().
 		SetXrayGraphScanParams(auditCmd.CreateXrayGraphScanParams()).
 		SetWorkingDirs(auditCmd.workingDirs).
 		SetMinSeverityFilter(auditCmd.minSeverityFilter).
 		SetFixableOnly(auditCmd.fixableOnly)
 	auditParams.GraphBasicParams = auditCmd.GraphBasicParams
-	results, isMultipleRootProject, auditErr := GenericAudit(auditParams)
 
 	serverDetails, err := auditParams.ServerDetails()
 	if err != nil {
 		return err
 	}
+	xrayManager, err := commandsutils.CreateXrayServiceManager(serverDetails)
+	if err != nil {
+		return err
+	}
+	errGroup := new(errgroup.Group)
+	entitled, err := xrayManager.IsEntitled(xrutils.ApplicabilityFeatureId)
+	if err != nil {
+		return err
+	}
+	if entitled {
+		// Download (if needed) the analyzer manager in a background routine.
+		errGroup.Go(utils.DownloadAnalyzerManagerIfNeeded)
+	} else {
+		log.Info(`The ‘jf audit’ command also supports the ‘Contextual Analysis’ feature,
+		which is included as part of the ‘Advanced Security’ package, which hasn’t
```

Comment:
...which is included as part of the ‘Advanced Security’ package, which hasn’t
-->
...which is included as part of the ‘Advanced Security’ package.